bazel/google_cloud_cpp_deps.bzl

```diff
@@ -191,9 +191,9 @@ def google_cloud_cpp_deps():
     if "io_opentelemetry_cpp" not in native.existing_rules():
         http_archive(
             name = "io_opentelemetry_cpp",
-            strip_prefix = "opentelemetry-cpp-d5571916e18abbd994373cdb633be0815deda319",
+            strip_prefix = "opentelemetry-cpp-201b347588973de0288f66d804b2edb9c8b6cdb9",
             urls = [
-                "https://github.com/open-telemetry/opentelemetry-cpp/archive/d5571916e18abbd994373cdb633be0815deda319.tar.gz",
```

Comment:
@dbolduc Can we refer to a version and not a SHA here?

Reply:
No. We are waiting for a bazel fix to be included in their next release. See #10498.